src/components/search/index.js

```diff
@@ -86,18 +42,25 @@ export default function Search({ collapse }) {
   )
 
   return (
-    <div className="search-root" ref={ref}>
+    <div className="relative flex-grow flex items-center" ref={ref}>
       <InstantSearch
         searchClient={searchClient}
         indexName={indices[0].name}
         onSearchStateChange={({ query }) => setQuery(query)}
-        root={{ Root, props: { ref } }}
-        styles={{ position: "relative" }}
       >
         <Configure hitsPerPage={8} />
         <Input onFocus={() => setFocus(true)} {...{ collapse, focus }} />
```

Comment:
@jasonLaster `collapse` or `focus` is not used in the code, so maybe we can remove them.